### PR TITLE
[Case 4315] Resolves Particle Tab Color Picker visual inconstency

### PR DESCRIPTION
**Pull request by 1P-Cusack**
* Brings Particle Tab color pickers inline with Properties Tab color pickers which don't have the ok/submit button.
* Fixes the overall visual styling of color pickers within the Particle Tab which didn't obey the expected Color Picker layout.
* Fixes an exception within entitySelectionTool encountered during testing which interfered with entity selection.

**_NOTE:_**  This is a follow up to PR #12391 where the testing revealed the Particle Tab color pickers weren't visually consistent with the other pickers (see: https://github.com/highfidelity/hifi/pull/12391#issuecomment-365688774)

**Particle Tab Color Picker Before:**
case4315_vfxtabbefore

**Particle Tab Color Picker After:**
case4315_vfxtabafter

### CODE REVIEW

**Comment by 1P-Cusack** [comment]
** Testing **

Testing should be done in both Desktop & HMD modes.

Create or select an instance of a particle system and expand the Alpha Section.

Each color field should have the same visual styling as the color pickers shown within the Properties Tab and also function the same.  Functionality points are denoted below:

- When opened show the same color as the color picker thumbnail within the color preview section.
- At this point both the New Color Preview (on the left) and the Old Color Preview (on the right) sections should have the same color appearing to be one solid piece.
- When manipulating the color via the picker color or hue area the New Color Preview (on the left side) should update accordingly. The Old Color Preview Section (on the right) should remain the same color as it was originally.
- Clicking on the Old Color Preview section should reset the Entity's Color to its original color and the New Color Preview section should update accordingly.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7418/app_12472_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7418/HighFidelity-Beta-PR12472-2f7f19ca6a0552b234dc666030d7a552fbcb5a1f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7418/HighFidelity-Beta-PR12472-2f7f19ca6a0552b234dc666030d7a552fbcb5a1f.exe

**Comment by qlimnavarro** [comment]
Currently Testing.

**Comment by qlimnavarro** [comment]
Pass @conklin94122

**Comment by conklin94122** [comment]
@themelissabrown Needs milestone

File: scripts/system/libraries/entitySelectionTool.js [hunk 1]
```diff
@@ -1322,8 +1322,8 @@ SelectionDisplay = (function() {
                                        isActiveTool(handleScaleRTFCube) || isActiveTool(handleStretchXSphere) || 
                                        isActiveTool(handleStretchYSphere) || isActiveTool(handleStretchZSphere));
 
-        var showOutlineForZone = (SelectionManager.selections.length === 1 &&
-                                  SelectionManager.savedProperties[SelectionManager.selections[0]].type === "Zone");
+        var showOutlineForZone = (SelectionManager.selections.length === 1 && (SelectionManager.savedProperties.length >= 1 &&
```
Comment (misslivirose) [inline scripts/system/libraries/entitySelectionTool.js]: This is a change from a separate PR - merge artifact?

Comment (LaShonda1P) [reply scripts/system/libraries/entitySelectionTool.js]: @misslivirose:  This isn't a merge artifact, it's an individual commit within this PR Commit 1165fa3c.  At the time this was posted there wasn't a fix here.  I encountered issues during testing which blocked my ability to verify the behavior I was looking for and traced it here.  It looks like the PR you mentioned was merged to master prior to this and that triggered the conflict on this branch.  I can do a quick rebase for this branch and resolve the conflict.

Comment (misslivirose) [reply scripts/system/libraries/entitySelectionTool.js]: Ah okay, great, thank you!

**Comment by ZappoMan** [comment]
Note: @1P-Cusack this PR has a merge conflict and needs to be updated before it can be merged.

**Comment by 1P-Cusack** [comment]
@ZappoMan:  I spoke with @misslivirose earlier regarding it; this PR will be updated soon just needed to get to a good checkpoint with another case that I was working on.  Thanks for the ping in case I'd not seen it; I appreciate it!

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7646/app_12472_PR.apk

**Comment by 1P-Cusack** [comment]
@misslivirose, @ZappoMan:  This updated version of this branch based on master at tag RELEASE-7915 should be building soon.  The only difference in this update is the version of master this is based on & dropping the `entitySelectionTool.js` change as discussed in https://github.com/highfidelity/hifi/pull/12472#discussion_r171699356

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7646/HighFidelity-Beta-PR12472-fde1ca62620e162c4804e6da8ab01f3ff0aa6256.dmg

**Comment by misslivirose** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7726/app_12472_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7726/HighFidelity-Beta-PR12472-fde1ca62620e162c4804e6da8ab01f3ff0aa6256.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7726/HighFidelity-Beta-PR12472-fde1ca62620e162c4804e6da8ab01f3ff0aa6256.exe

**Pull request merged**